pkg/quotes/README.md

```diff
@@ -39,13 +40,73 @@ last_price = price
 
 ## Index Price
 
-Used mainly in risk management for portfolio evaluation:
+Index Price is used mainly in risk management for portfolio evaluation, and is aggregated from multiple sources.
 
 ```
-index_price = EMA20(price x ( trade_size x weight / active_weights ))
-# active_weight being the sum of weight where this market exists (ex: KuCoin:5 + uniswap:50)
-# EMA20 is likely 20 at 1 min scale
+index_price = priceWeightEMA/weightEMA
+
+priceWeightEMA = EMA20((Volume)*(Price)*(DriverWeight/activeWeights))
+
+weightEMA = EMA20(Volume*(DriverWeight/activeWeights))
+```
+```
+activeWeights is the sum of weight where this market is active (ex: KuCoin:5 + uniswap:50)
+```
+
+This formula calculates a weighted EMA price, using trade volume, and active driver weight as additional weights.
+
+EMA20 is calculated based on the last 20 trades. If on the startup there is no initial data, the values of the first trade received are used as the initial EMA.
+
+Let's assume 5 trades have been received:
+
 ```
+{source, price, amount}
+{binance, 41000, 0.3}
+{binance, 42500, 0.5}
+{uniswap, 55000, 0.6}
+{uniswap, 50000, 0.4}
+{binance, 40000, 1.0}
+```
+
+In the first example, drivers' weights are equal: ```2``` for the Binance driver, and ```2``` for the Uniswap.
```

Comment:
These examples are confusing as the reference to the test file is way below.
can you explain why we assume the weight instead of calculating it?

Reply:
@dimast-x we need to show calculation not only input and results
@pkucherenk0 weight for each source is static